core/src/main/kotlin/fr/sncf/osrd/api/api_v2/standalone_sim/SimulationRequest.kt

```diff
@@ -88,6 +88,7 @@ class SimulationPath(
     val blocks: List<String>,
     val routes: List<String>,
     @Json(name = "track_section_ranges") val trackSectionRanges: List<TrackRange>,
+    @Json(name = "path_item_positions") val pathItemPositions: List<Offset<Path>>
```

Comment:
We talked and decided a while ago it'd be clearer in the apis if we used `Offset<TravelledPath>`. There's a couple over places where we need to do this change, so just leave it as it is for now, I'll do it later on in another PR :)